packages/utilities/src/dom/getDocument.ts

```diff
@@ -7,10 +10,16 @@ import { _isSSR } from './setSSR';
  *
  * @public
  */
-export function getDocument(rootElement?: HTMLElement | null): Document | undefined {
+export function getDocument(rootElement?: HTMLElement | Component | null): Document | undefined {
   if (_isSSR || typeof document === 'undefined') {
     return undefined;
   } else {
-    return rootElement && rootElement.ownerDocument ? rootElement.ownerDocument : document;
+    if (rootElement && !(rootElement as Element).ownerDocument) {
+      // tslint:disable-next-line:no-any
+      rootElement = findDOMNode(rootElement) as any;
```

Comment:
Same as below, also why is this typed as any but the one in getWindow is typed as Element?